code/controllers/subsystem.dm

```diff
@@ -163,8 +163,21 @@
 	initialized = TRUE
 	var/time = (REALTIMEOFDAY - start_timeofday) / 10
 	var/msg = "Initialized [name] subsystem within [time] second[time == 1 ? "" : "s"]!"
-	to_chat(world, "<span class='boldannounce'>[msg]</span>")
-	log_world(msg)
+
+	//The formatting of outputs depends on the time taken
+	//Longer than one second, its in red
+	//The formatting of outputs depends on the time taken
+	//Longer than one second, its in red
+	if (time > 1)
+		//log_world(msg)
+		report_progress("<span class='boldannounce'>[msg]</span>")
```

Comment:
Will this not spam up the discord if this is run on live? Or am I misreading report_progress()

Reply:
the total number of logs is less than before